docker/pravega/scripts/common.sh

```diff
@@ -36,9 +36,12 @@ add_system_property_ecs_config_uri() {
 
 # Add ECS certificates into Java truststore
 add_certs_into_truststore() {
-    CERTS=/etc/ssl/certs/java/ecs-certs/*
+    password=`cat /etc/ssl/certs/java/ecs-certs/JAVA_TRUST_STORE_PASSWORD`
+    trustStore=`cat /etc/ssl/certs/java/ecs-certs/JAVA_TRUST_STORE_PATH`
```

Comment:
Why do we allow to specify a path to the truststore if we anyway use the default truststore in Segment Store? 

As a user I can provide a different path that would be pointing to some custom keystore inside the container but that keystore wouldn't be loaded by SSS

Reply:
We can use the Kubernetes secrets mechanism to get a user-specified truststore/password into the pod at deployment time. That's what we do for Controller and Segment store TLS material (see https://github.com/pravega/pravega-operator/blob/master/doc/tls.md)

Reply:
Yes, now the ECS TLS materials are to be prepared as k8s secret by admin, and the Pravega-Operator mounts the secret into secret-volume, in the same way we mount other TLS secrets.

The ECS TLS config will be transparent to ECS client side config.